Page/PageServiceManager.php

```diff
@@ -126,7 +126,8 @@ protected function createResponse(PageInterface $page)
     {
         if ($page->getTarget()) {
             $page->addHeader('Location', $this->router->generate($page->getTarget()));
-            $response = new Response('', 302, $page->getHeaders() ?: array());
+            $redirectType = $page->getTargetRedirectType() ? $page->getTargetRedirectType() : 302;
```

Comment:
Constant not available in symfony 2.3 :(
https://github.com/symfony/http-foundation/blob/2.3/Response.php

https://travis-ci.org/sonata-project/SonataPageBundle/jobs/266601244